docs/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md

```diff
@@ -4,17 +4,677 @@ title: Connector SDK
 description: Introduction to the Connector SDK.
 ---
 
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
 :::note
 The **connector SDK** is in developer preview and subject to breaking changes. Use at your own risk.
 :::
 
-The **connector SDK** allows you to develop your own **connectors**. With the help of the SDK you can perform common operations, such as:
+The **connector SDK** allows you to [develop custom connectors](#creating-a-custom-connector)
+using Java code. You can focus on the logic of the connector, conveniently test it locally, and
+reuse its runtime logic in multiple environments. The SDK achieves this by abstracting from
+Camunda Platform 8 internals that usually come with
+[job workers](../../../concepts/job-workers.md).
+
+The SDK provides API for common connector operations, such as:
```

Comment:
We should replace with one of the below options (unsure which one though).

"The SDK provides an API for common connector operations, such as:" OR
"The SDK provides APIs for common connector operations, such as:"

Reply:
```suggestion
The SDK provides APIs for common connector operations, such as:
```